source/features/one-click-review-submission.tsx

```diff
@@ -47,14 +50,15 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 			<button
 				type="submit"
 				name="pull_request_review[event]"
-				// The buttons are no longer inside the form itself; this links the form
+				// Old version of GH don't nest the submit button inside the form, so must be linked manually. Issue #6963.
 				form={formAttribute}
 				value={radio.value}
 				className={classes.join(' ')}
 				aria-label={tooltip!}
 				disabled={radio.disabled}
 			>
-				{radio.nextSibling}
+				{/* Old || new. Backwards-compat label content queries. Issue #6963. */}
+				{radio.nextSibling?.textContent?.trim() || labelNode?.textContent}
```

Comment:
So this label fix works by:

- Trying to use the old approach _first_, 
- Failing properly if `nextSibling` doesn't contain any text (in the new structure, it was containing a line break hence the trim)
- Falling back to the new way of doing labelling which only has the single-word labels, where the description is now somewhere else